### PR TITLE
Create Open Telemetry docs

### PR DESCRIPTION
**Pull request by DataBoyTX**
I copied this from Percy's wiki page, removing any internal references to servers, dev envs, etc.

### CODE REVIEW

**Review by aucahuasi — COMMENTED** [review]
I think we can rename the doc to `telemetry.md` instead of `Open-Telemetry`.

**Comment by lmeyerov** [comment]
* i renamed to `telemetry.md`
* I made a quick pass on text

@aucahuasi :

1. I thought commands need to run via `./release ...` to work the new services, not `docker compsoe`

2. Maybe we need to experiment & chat (so after this PR), but to turn on the local UIs, we should show how to setup Caddy for limiting access to admins, and pointers to how to look at logs vs metrics vs traces. Likely can enlist @dess890  for writing up the latter once we've got smooth?

File: docs/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+JAEGER_PUB_COMPOSE_FILE=compose/jaeger-pub.yml
+```
+
+## Usage
+
+By default, the telemetry services are disabled, so using `docker compose` commands will not trigger telemetry collection, forwarding, nor telemetry UIs
+
+### Turning on
+
+The `docker compose` will automatically start those services when `ENABLE_OPEN_TELEMETRY=true`.  Once the services are online, we can access these useful links for development and operations:
```
Comment (aucahuasi) [inline docs/telemetry.md:59]: change `docker compose` for `release`.

**Comment by lmeyerov** [comment]
fixed, merging

@aucahuasi next time feel free to edit directly in cases like this , you can hit `.` in github and it'll go into edit mode

**Review by lmeyerov — APPROVED** [review]

**Pull request merged**